test/old/testdir/test_cmdline.vim

```diff
@@ -2055,6 +2055,7 @@ endfunc
 " Test error: "E135: *Filter* Autocommands must not change current buffer"
 func Test_cmd_bang_E135()
   new
+  set shelltemp
```

Comment:
Thinking about this again, since `set shelltemp` is now added to `testdir/setup.vim`, this change is no longer needed. And if the next step is to remove the `'shelltemp'` option, a different and more careful change will be needed for this test anyway.

Reply:
Good point, will undo those changes.